src/commcare_cloud/ansible/roles/supervisor/tasks/main.yml

```diff
@@ -81,5 +81,5 @@
 
 - name: start supervisord
   become: yes
-  service: name=supervisord state=started
+  service: name=supervisord enabled=yes
```

Comment:
Does `enabled=yes` also imply `state=started`? I.e. in addition to making it start on boot, does `enabled=yes` make it start _now_? I tried to find where it said that in https://docs.ansible.com/ansible/latest/modules/service_module.html but maybe I just missed it.

Reply:
ah, good point. I didn't test if that happened. updated.